### PR TITLE
Generate command to build pylibcudf

### PR DESCRIPTION
**Pull request by lithomas1**
This makes the build-pylibcudf-python command and makes build-cudf depend on it.

Potentially needs admin-merge when changes look good?
cc @vyasr

### CODE REVIEW

File: features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml [hunk 1]
```diff
@@ -78,9 +78,13 @@ repos:
...
       depends: [cudf]
   python:
+    - name: pylibcudf
+      sub_dir: python/pylibcudf
+      depends: [cudf]
+      args: {cmake: -DFIND_CUDF_CPP=ON, install: *rapids_build_backend_args}
     - name: cudf
       sub_dir: python/cudf
-      depends: [cudf]
+      depends: [cudf, pylibcudf]
```
Comment (trxcllnt) [inline features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml:87]: Don't need to modify this line, as this indicates a dependency on the C++ `cudf` library:

```suggestion
      depends: [cudf]
```

**Comment by lithomas1** [comment]
@vyasr @jameslamb 

Can I also get a review here since y'all are already reviewing the other PR?

Thanks.

File: features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml [hunk 2]
```diff
@@ -78,6 +78,10 @@ repos:
       sub_dir: cpp/libcudf_kafka
       depends: [cudf]
   python:
+    - name: pylibcudf
+      sub_dir: python/pylibcudf
+      depends: [cudf]
+      args: {cmake: -DFIND_CUDF_CPP=ON, install: *rapids_build_backend_args}
     - name: cudf
       sub_dir: python/cudf
       depends: [cudf]
```
Comment (vyasr) [inline features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml:87]: You'll want to change this to also depend on pylibcudf.

**Review by vyasr — CHANGES_REQUESTED** [review]

File: features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml (hunk 2 above)
Comment (bdice) [reply features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml:87]: https://github.com/rapidsai/devcontainers/pull/375#discussion_r1697682716

Comment (lithomas1) [reply features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml:87]: Yeah, I think this doesn't require any changes.

I believe build-cudf does the blocks in order on the python side.
(i.e. cudf-kafka does not depend on cudf-python here even though it needs to compile against the cudf-python Cython)

Comment (vyasr) [reply features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml:87]: Oh you're totally right, I forgot how this gets used. This is just to set up the CMake arguments for build dependencies, Python builds just rely on the ordering of the manifest.

**Comment by vyasr** [comment]
Once CI is passing on https://github.com/rapidsai/cudf/pull/16299 for every other job, we can admin merge this one, redeploy, and rerun the devcontainer tests on that PR.

**Review by jameslamb — APPROVED** [review]
agree with @vyasr 's comment about the order of merging for this.

**Review by vyasr — APPROVED** [review]

**Pull request merged**